### ISSUE TITLE
lmrobdet.control efficiency

### ISSUE THREAD

**Issue by harrysouthworth** [msalibian/RobStatTM#6]
This might not be an "issue" but I don't know where else to ask.

Section 5.9.4 of the book recommends to use 99% efficiency.

lmrobdet.controll defaults to efficiency = 0.95.

Should "99%" in the book be "95%", should efficiency = 0.95 be efficiency = 0.99,  or is the difference deliberate?

Many thanks,
Harry

**Comment by msalibian** [comment]
Thanks for the note. The explanation for the discrepancy is as follows. We discovered after publication of the book that, based on empirical evidence, use of a 99% default normal distribution efficiency sacrificed too much robustness. Thus we changed to a 95% default.  Note that the default rho function is also different ("mopt" instead of "opt"). This is because the opt weight function does not satisfy a condition needed to insure the convergence of the final IRWLS algorithm, and the mopt has a change in the weight function shape that was created to insure IRWLS convergence.  Details on the opt versus mopt weight functions, can be found in Konis and Martin (2021). "Optimal Bias Robust Regression Psi and Rho Revisited", available at SSRN [https://ssrn.com/abstract=3902862](https://ssrn.com/abstract=3902862).
